### DISCUSSION TITLE
Study from mobile

### DISCUSSION THREAD

**Discussion by wenahyuck** [Cyfrin/foundry-full-course-cu#2781] [Q&A]
Guys is there a way I can learn code from my mobile through GitHub without watching the video, just seeing the code and the concept related to it? Can the code be accessed from here?

**Comment by EngrPips** [comment]
Hello @wenahyuck, why do you want to learn the course content without watching the video? And actually there are written lessons on updraft if you prefer written lessons.

**Reply by wenahyuck** [reply]
Hello sir, I'm talking about just going through the code in mobile if we have spare time and understanding it.

**Reply by EngrPips** [reply]
If you go to the `code` tab of this Github you can see the link to the codebase for each section there

**Reply by wenahyuck** [reply]
Where ser, cannot see the code tab here.

**Reply by EngrPips** [reply]
Shown in the images below 

<img width="1440" alt="Screenshot 2024-11-01 at 20 17 10" src="https://github.com/user-attachments/assets/a134e8c0-f96c-4f34-bc32-fb103aaa8c77">

<img width="1440" alt="Screenshot 2024-11-01 at 20 21 45" src="https://github.com/user-attachments/assets/69491dea-5a52-46ab-bace-b82d652b373d">

**Reply by wenahyuck** [reply]
Hello ser, is that code tab visible in your mobile?

**Reply by wenahyuck** [reply]
Sorry ser, got the code tab. Thanks so much.

**Reply by EngrPips** [reply]
You are welcome, Friend. Keep crushing it.
